sources/packages/api/src/services/msfaa-number/msfaa-number.service.ts

```diff
@@ -93,4 +99,69 @@ export class MSFAANumberService extends RecordDataModelService<MSFAANumber> {
       dayjs(endDate).diff(startDate, "days") < MAX_MFSAA_VALID_DAYS
     );
   }
+
+  /**
+   * Fetches the MSFAA number records which are not sent for request.
+   * This can be retrived by checking for date_requested column as null
+   * in the MSFAANumber table.
+   * @returns the records of the MSFAANumber table.
+   */
+  async getPendingMSFAARequest(
+    offeringIntensity: string,
+  ): Promise<MSFAANumber[]> {
+    return this.repo
+      .createQueryBuilder("msfaaNumber")
+      .select([
+        "msfaaNumber.id",
+        "students.id",
+        "referenceApplication.id",
+        "msfaaNumber.msfaaNumber",
+        "students.sin",
+        "institutionLocation.institutionCode",
+        "students.birthdate",
+        //"students.maritalStatus",
```

Comment:
commented line of code.

Reply:
https://github.com/bcgov/SIMS/pull/827#discussion_r740505848 - For now we are having it commented, this will be changed when we decide on where we are fetching the marital status in the future stories. I believe I have mentioned it in my comments.

Reply:
Okay. seeing this it made me feel that we commented out a property which we didn't need to be returned at last minute. Got it now.

Reply:
Whatever it is it would be good to add a TODO comment. This will be marked on Sonar Cloud and we will never forget it 😉